src/libxrpl/protocol/LedgerFormats.cpp

```diff
@@ -364,6 +364,36 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,   soeREQUIRED}
         },
         commonFields);
+		
+	add(jss::MPTokenIssuance,
+        ltMPTOKEN_ISSUANCE,
+        {
+            {sfIssuer,               soeREQUIRED},
+            {sfSequence,             soeREQUIRED},
+            {sfTransferFee,          soeDEFAULT},
+            {sfOwnerNode,            soeREQUIRED},
+            {sfAssetScale,           soeDEFAULT},
+            {sfMaximumAmount,        soeOPTIONAL},
+            {sfOutstandingAmount,    soeREQUIRED},
+            {sfLockedAmount,         soeDEFAULT},
+            {sfMPTokenMetadata,      soeOPTIONAL},
+            {sfPreviousTxnID,        soeREQUIRED},
+            {sfPreviousTxnLgrSeq,    soeREQUIRED}
+        },
+        commonFields);
+
+    add(jss::MPToken,
+        ltMPTOKEN,
+        {
+            {sfAccount,           soeREQUIRED},
+            {sfMPTokenIssuanceID, soeREQUIRED},
+            {sfMPTAmount, soeDEFAULT},
```

Comment:
`MPTAmount` is marked required in the spec.

Reply:
IMHO, the spec should be updated for `MPTAmount`. There are potentially going to be a lot of MPT holders that don't hold a balance. Leaving this as `soeDEFAULT` will save 64-bits for every one of them.

Reply:
Yes this is a mistake in the spec. We originally intended to implement it as required. But as @ximinez  pointed out, we would want to mark it as default to save space